src/main/webapp/app/lecture/lecture-unit/lecture-unit-management/attachment-units/attachment-units.component.ts

```
@@ -117,7 +117,7 @@ export class AttachmentUnitsComponent implements OnInit {
             )
             .subscribe({
                 next: (res) => {
-                    this.units = res.body!.units;
+                    this.units = res.body!.units || this.units;
```

Biome

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)